src/systemTest/java/randoop/main/RandoopSystemTest.java

```diff
@@ -394,18 +409,25 @@ public void runJDKTest() {
             options,
             "java7.util7.ArrayList.addAll(int, java7.util7.Collection) ignore",
             "java7.util7.ArrayList.addAll(java7.util7.Collection) ignore",
-            "java7.util7.ArrayList.fastRemove(int) exclude",
+            "java7.util7.ArrayList.fastRemove(int) ignore",
```

Comment:
Try not to change to "ignore".  Doing so makes the tests weaker, and makes it harder for subsequent programmers to see whether they have improved the situation or not.  Deleting the line makes it required.  (Is that a bad idea in this case?)